index.html

```diff
@@ -0,0 +1,90 @@
+  <head>
+    <meta charset="UTF-8">
+    <meta name="description" content="Microverse portifolio clone activity.">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/styles.css"/>
+        <title>Microverse Portifolio - Template 3</title>
+  </head>
+  <body>
+    <header>
+    <section id="mobile-expanded-menu" class="mobile-expanded-menu disabled">
+      <nav id="expanded-nav" class="expanded-nav flex-container-column">
+        <a class="close-btn" href="#"><img id="close-btn" class="close-img" src="./css/img/mobile_menu/Vector.svg" alt=""></a>
+          <ul id="expanded-menu-list" class="expanded-menu-list">
+          <li id="expanded-menu-item" class="expanded-menu-item  project-title-custom-font"><a id="expanded-section-link" href="#portfolio">Portifolio</a></li>
+          <li class="expanded-menu-item  project-title-custom-font"><a href="#about">About</a></li>
+          <li class="expanded-menu-item  project-title-custom-font"><a href="#contact">Contact me</a></li>
+        </ul>
+      </nav>
+    </section>
+    </header>
+    <header>
+      <div id="toolbar-container" class="toolbar-container flex-container-row">
+        <div class="toolbar logo">
+          <a class="logo-link" href="#" title="Welcome HomePage link">
+              <h3 class="logo-text default-custom-font">My Logo</h3>
+          </a>
+        </div>
+        <nav class="toolbar menu">
+          <div class="menu-trigram">
+            <a id="menu-btn" class="menu-link" href="#" title="Open Menu link"></a>
+          </div>
+          <ul class="toolbar-menu-desktop flex-container-row disabled">
+            <li>
+              <a href="#portfolio" class="desktop-menu-item default-custom-font">
+                Portfolio
+              </a>
+            </li>
+            <li>
+              <a href="#about" class="desktop-menu-item default-custom-font">
+                About
+              </a>
+            </li>
+            <li>
+              <a href="#contact" class="desktop-menu-item default-custom-font">
+                Contact
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <section id="headline-container" class="headline-container flex-container-column">
+      <div class="headline primary-text flex-container-column">
+        <h1 class="primary-text first-color primary-text-custom-font">Hey there. I'm Habtamu Alemayehu</h1>
+        <h2 class="primary-text second-color primary-text-custom-font">I'm a software developer</h2>
+      </div>
+      <div class="headline supporting-text">
+        <p class="sup-text-paragraph default-custom-font">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </div>
+      <div class="logo-text first-color">  
+        LETTE'S CONNECT
+      </div>
```

Comment:
Very well done in this project , only this title doesn't exist in the template just remove it and it has a typo always make sure that your text is correct as this is a sign of professionalism

![figma temp2](https://user-images.githubusercontent.com/59707859/227528192-2f0b7e05-beb3-4bc7-bb12-e51a63a4f4e6.PNG)

![student temp3](https://user-images.githubusercontent.com/59707859/227528274-994a54b6-32be-41a9-993e-462e96e72826.PNG)